README.md

```diff
@@ -1,33 +1,32 @@
-# Project
+# Azure Terraform Verified Modules
 
-> This repo has been populated by an initial template to help get you started. Please
-> make sure to update the content to build a great experience for community-building.
+> This repo contains the source code of all currently available Terraform modules verified by Microsoft Azure. All verified modules will meet the quality pipeline established by Microsoft Azure team, which guarantee their high quality.    
 
-As the maintainer of this project, please make a few updates:
+## Modules
+Below is a table containing all Terraform modules verified by Microsoft Azure. Each module version badge shows the latest version of the corresponding module. You may click on a version badge to check all available versions for a specific module. Each minimum Terraform version shows the minimum entry requirements for the version of Terraform. Using the verified modules below the required Terraform version might cause potential inconsistency and disruption. 
+> For those with parallel modules, we strongly recommend you transfer your existing infrastructure to the new verified modules, as old parallel modules will no longer be updated. 
```

Comment:
Lets remove references to parallel modules. I think that will just add confusion.